VideoAPI/Video.API/Mappings/ConferenceToclosedConferenceMapper.cs

```diff
@@ -0,0 +1,16 @@
+using VideoApi.Contract.Responses;
+using VideoApi.Domain;
+
+namespace Video.API.Mappings
```

Comment:
Check the filename. There's a lowercase c for closed, should be upper-case

Reply:
corrected the name.